src/universal_transfer_operator/integrations/fivetran/connector/__init__.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+import importlib
+
+from universal_transfer_operator.datasets.base import Dataset
+from universal_transfer_operator.integrations.fivetran.connector.base import FivetranConnector
+from universal_transfer_operator.utils import get_class_name
+
+CUSTOM_FIVETRAN_CONNECTOR_TYPE_TO_MODULE_PATH = {
+    "s3": "universal_transfer_operator.integrations.fivetran.connector.aws.s3",
+    "aws": "universal_transfer_operator.integrations.fivetran.connector.aws.s3",
+}
+
+
+def get_fivetran_connector(
```

Comment:
Where is this method getting used?

Reply:
It is being used here: https://github.com/astronomer/apache-airflow-provider-transfers/blob/fivetran-setup/src/universal_transfer_operator/integrations/fivetran/fivetran.py#L16 and https://github.com/astronomer/apache-airflow-provider-transfers/blob/fivetran-setup/src/universal_transfer_operator/integrations/fivetran/fivetran.py#L461